src/pages/Landing/Hero.tsx

```diff
@@ -75,7 +76,8 @@ export const Hero = () => {
                   color="brand"
                   className="my20"
                 >
-                  Become a validator and help secure the eth2 network.
+                  Become a validator and help secure the eth2{' '}
+                  {IS_MAINNET ? ` mainnet` : ` ${ETH2_NETWORK_NAME} testnet`}.
```

Comment:
IMHO we can highlight `Altona testnet` again to warn the users that they are playing on testnet.
Or even note that it's testnet on the header banner, e.g., `Eth2 Launch Pad for Altona testnet`.
![image](https://user-images.githubusercontent.com/9263930/86504114-897d3280-bde7-11ea-896c-6e31032dc867.png)

Reply:
I agree that more clarification is a good idea. I've set up #93 to address this and add more clarification on the matter..